### DISCUSSION TITLE
如何进行 pixiv 随机检索

### DISCUSSION THREAD

**Discussion by CCXXXI** [koishijs/koishi-plugin-booru#90] [Q&A] [answered]
我看到文档中有：

https://github.com/koishijs/koishi-plugin-booru/blob/26ef0e81aeafb3dd17cbcb8d1700b5af4406bca8/docs/zh-CN/index.md?plain=1#L16

但是使用 pixiv 作为图片源时，对于同样的关键词，总是会给出同一张图片。

如何从所有检索结果中随机获取一张？

**Comment by MaikoTan** [comment, marked as answer]
The pixiv source didn't support randomly search currently, would be added in the next version.
